### PR TITLE
Fixing SEO stuffs + add route to sitemap

### PR DESCRIPTION
**Pull request by shrix1**
### Description

- Fixed some SEO stuffs

### Related Issue

- Helps fixing : https://github.com/trypear/pear-landing-page/issues/77

### Changes Made

- Add `/signup` and `/signin` in sitemap 
- Fixing html tag for SEO optimization (Each page should only have one `h1` tag that should be important text (so screenreader can easily identify important sentence ))
- Remove `yarn build` from `pre-commit` script (its useless to run on every commit (takes lot of time))

### Screenshots

- no

### Checklist

- [x] I have tagged the issue in this PR.
- [x] I have attached necessary screenshots.
- [x] I have provided a short description of the PR.
- [x] I ran `yarn build` and build is successful
- [x] My code follows the style guidelines of this project.
- [x] I have added necessary documentation (if applicable)

### CODE REVIEW

**Comment by shrix1** [comment]
If other routes like `reset-password` , etc .. functionality done means i can add to sitemap

File: components/features.tsx [hunk 1]
```diff
@@ -77,12 +77,12 @@ function SingleFeature(props: SingleFeatureProps) {
   return (
     <div className="m-4 flex flex-col items-start min-[425px]:m-8 sm:m-12 lg:m-24 lg:max-w-full lg:flex-row lg:justify-center">
       <div className="justify mb-8 flex flex-col content-center pr-8 text-left lg:mb-0 lg:ml-1.5 lg:w-[450px]">
-        <h1
+        <h4
```
Comment (Fryingpannn) [inline components/features.tsx:80]: Are we still using these?

**Comment by Fryingpannn** [comment]
> If other routes like `/reset-password` , etc .. functionality done means i can add to sitemap @Fryingpannn @ICEPrey

Yes those are done!

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,3 +1,2 @@
 yarn format
 yarn lint
-yarn build
```
Comment (Fryingpannn) [inline .husky/pre-commit:3]: let's keep this in

File: components/features.tsx (hunk 1 above)
Comment (shrix1) [reply components/features.tsx:80]: yess

File: .husky/pre-commit (hunk 2 above)
Comment (shrix1) [reply .husky/pre-commit:3]: okay

**Pull request merged**